src/mono/mono/mini/CMakeLists.txt

```diff
@@ -30,7 +30,7 @@ set(OS_LIBS "-framework CoreFoundation" "-lobjc" "-lc++")
 elseif(HOST_ANDROID)
 set(OS_LIBS m dl log)
 elseif(HOST_LINUX)
-set(OS_LIBS pthread m dl)
+set(OS_LIBS pthread m dl atomic)
```

Comment:
Hmm this needs to be turned into a check.
Some C libraries require this, others don't have it